editor/index.js

```diff
@@ -46,22 +46,48 @@ window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
 } );
 
 /**
+ * Reinitializes the editor after the user chooses to reboot the editor after
+ * an unhandled error occurs, replacing previously mounted editor element using
+ * an initial state from prior to the crash.
+ *
+ * @param {Element} target       DOM node in which editor is rendered
+ * @param {*}       initialState Initial editor state to hydrate
+ */
+export function recreateEditorInstance( target, initialState ) {
+	unmountComponentAtNode( target );
+
+	const reboot = recreateEditorInstance.bind( null, target );
+
+	render(
+		<EditorProvider initialState={ initialState }>
```

Comment:
Can you confirm that we can safely omit „settings” prop here?

Reply:
Hmm, on second glance, I think this may be needed. I had assumed this was being used to initialize state and therefore would be preserved in the reboot, but in fact this is part of a separate context.